src/deadline/keyshot_submitter/Submit to AWS Deadline Cloud.py

```diff
@@ -0,0 +1,417 @@
+    with open(os.path.join(directory, filename), "w") as file:
+        file.write(json.dumps(contents))
+
+
+def load_sticky_settings(scene_filename: str) -> Optional[dict]:
+    sticky_settings_filename = Path(scene_filename).with_suffix(RENDER_SUBMITTER_SETTINGS_FILE_EXT)
+    if sticky_settings_filename.exists() and sticky_settings_filename.is_file():
+        try:
+            with open(sticky_settings_filename, encoding="utf8") as f:
+                return json.load(f)
+        except (OSError, json.JSONDecodeError):
+            # Fall back to defaults if there's an error loading sticky settings
+            import traceback
+
+            traceback.print_exc()
+            print(
+                f"WARNING: Failed to load {sticky_settings_filename}. Reverting to the default settings."
+            )
+    return None
+
+
+def save_sticky_settings(scene_file: str, settings: Settings):
+    sticky_settings_filename = Path(scene_file).with_suffix(RENDER_SUBMITTER_SETTINGS_FILE_EXT)
+    with open(sticky_settings_filename, "w", encoding="utf8") as f:
+        json.dump(settings.output_sticky_settings(), f, indent=2)
+
+
+def gui_submit(bundle_directory: str) -> Optional[dict[str, Any]]:
+    try:
+        if platform.system() == "Darwin" or platform.system() == "Linux":
+            # Execute the command using an bash in interactive mode so it loads loads the bash profile to set
+            # the PATH correctly. Attempting to run `deadline` directly will probably fail since Keyshot's default
+            # PATH likely doesn't include the Deadline client.
+            shell_executable = os.environ.get("SHELL", "/bin/bash")
+            result = subprocess.run(
+                [
+                    shell_executable,
+                    "-i",
+                    "-c",
+                    f'echo "START_DEADLINE_OUTPUT";deadline bundle gui-submit {shlex.quote(bundle_directory)} --output json',
+                ],
+                check=True,
+                capture_output=True,
+                text=True,
+            )
+            # Hack to ignore any output from the bash profile script
+            output = result.stdout.split("START_DEADLINE_OUTPUT")[-1]
+        else:
+            result = subprocess.run(
+                [
+                    "deadline",
+                    "bundle",
+                    "gui-submit",
+                    str(bundle_directory),
+                    "--output",
+                    "json",
+                ],
+                check=True,
+                capture_output=True,
+                text=True,
+            )
+            output = result.stdout
+    except subprocess.CalledProcessError as e:
+        raise RuntimeError(f"AWS Deadline Cloud KeyShot submitter could not open: {e.stderr}")
+    try:
+        return json.loads(output)
+    except json.JSONDecodeError as e:
+        print(f"Error parsing settings file: {e}")
```

Comment:
copy/paste? Should indicate it failed to read deadline-cli output as json

Reply:
Updated!